query/src/users/user_mgr.rs

```diff
@@ -36,6 +36,11 @@ impl UserApiProvider {
                         UserPrivilegeSet::available_privileges_on_global(),
                     );
                     user_info.option.set_all_flag();
+                } else if user.is_root() {
```

Comment:
@BohuTANG should we accept `default@localhost` too for the convenience of clickhouse client? 

choices:

1.  accept,  `default@localhost` and `root@localhost` works as root for all handlers
2. deny `default@localhost`,  prompt the clickhouse user to use `root` 
3.  accept  `default@localhost`  only for clickhouse handler

Reply:
I think 2 or 3 is ok, which one is easy to do?

Reply:
> which one is easy to do?

almost the same： just need a small refactor to pass an enum indicating the handler for the auth-related methods,   (for 2 only if we need `prompt the clickhouse user to use root` )

for 2 also need to deny create user called `default  `

Reply:
Got, looks 1 is the better way, we allow `default` and `root` users only from the local host.
BTW, no special considerations here, this is to restrict these default users(`default`/`root`) to local login only.

Reply:
@chowc so what about change `is_root()` to include `default` and use it  for `if`

Reply:
So we accept 'default' and 'root' username as root user and only allow them to login in localhost? Am i right? 
Also empty login name will be denied.

Reply:
> @chowc so what about change `is_root()` to include `default` and use it for `if`

ok, I will update it.